loaders/playgroundLoader-en.js

```diff
@@ -0,0 +1,3 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const loader=require('./playgroundLoader')
```

Comment:
Better format this line

Reply:
esline disallow  `require`, but it is necessary for nodejs environment

Reply:
format means adding blank space